src/components/data/DataTable/DataTable.tsx

```diff
@@ -114,12 +114,81 @@ type DataTableState = {
  * DataTable component enables fetching data using predefined function and showing tabular data in a simple manner.
  *
  * ## Features
- * - data pagination
+ * - data fetch with pagination
  * - selectable rows
  * - expandable rows
  * - data sorting by columns
  * - filtering data
  *
+ * ## Data Fetch Details
+ * Fetch data is a function that accepts gridParam parameters
```

Comment:
When I read it for the first time it was not clear for me to which function are you referring to, so wdyt about:
```suggestion
 * `fetchData` prop is a function that accepts grid parameters:
```
?